sha256sums.txt

```diff
@@ -1,38 +1,39 @@
-99cc931e11dbef6e41e9376f18be62fc90fe4be9c541eac1b30a3455b3d655f3  alembic-1.0.2-py2.py3-none-any.whl
+ad1e93816f37f8445bb12dfe412579d2237818c9bbf34054a06e00d5f430c700  alembic-1.0.2-py2.py3-none-any.whl
 04bcb970ca8659c3607ddd8ffd86cc9d6a99661c9bc590955e8813c66bfa582b  alembic-1.0.2.tar.gz
-fc8c8e0b587d00f38986bc161f4496e000acea033fe2ce25f4f5bffa9ae53a7c  arrow-0.12.1-py2.py3-none-any.whl
+e2742eb33011f7aff1d5f27501d802680b6627939f20ff2ef56f050c14af59fa  arrow-0.12.1-py2.py3-none-any.whl
 a558d3b7b6ce7ffc74206a86c147052de23d3d4ef0e17c210dd478c53575c4cd  arrow-0.12.1.tar.gz
-173b19dd31ca7faa50d1fcc0eaf30f5e32e8e99e17d8c7fd4cfc8bc8d94e18a6  certifi-2018.10.15-py2.py3-none-any.whl
+2d5538b9d89b3a9cb423b8f08417c657856cab733d560067e864157f71b1c5aa  certifi-2018.10.15-py2.py3-none-any.whl
 6d58c986d22b038c8c0df30d639f23a3e6d172a05c3583e766f4c0b785c0986a  certifi-2018.10.15.tar.gz
-f5632e583a4f61f1e16d0cc98127d241fb11c3c6ddfddee159307d4215186837  chardet-3.0.4-py2.py3-none-any.whl
+e5cf39014befb85add77118fdc946f0a3387df7840235ba5d20fd6e3a672410a  chardet-3.0.4-py2.py3-none-any.whl
 84ab92ed1c4d4f16916e05906b6b75a6c0fb5db821cc65e70cbd64a3e2a5eaae  chardet-3.0.4.tar.gz
-1855003e64dcb934556ad79994ba1a3a852da337e353d84d3b4ef75031913451  furl-2.0.0-py2.py3-none-any.whl
+9f50360f6e4a0f1d0a35fb4997878e7186a73331f0fde5f6fc9b1bb9f006e6cc  furl-2.0.0-py2.py3-none-any.whl
 fdcaedc1fb19a63d7d875b0105b0a5b496dd0989330d454a42bcb401fa5454ec  furl-2.0.0.tar.gz
-491f674364ba3232ed1eb4c1eb7407887f62cef6c300aad7df6e01acd88ffb25  idna-2.7-py2.py3-none-any.whl
+a967a752dba5492be34f30f8633cee512d2ec923fe3a2e201d87353b540d2955  idna-2.7-py2.py3-none-any.whl
 684a38a6f903c1d71d6d5fac066b58d7768af4de2b832e426ec79c30daa94a16  idna-2.7.tar.gz
-614c22fe1a5b0a3f46f6c5c43ff2e6795e4e784328d559ec9dc49db0f06b3a75  Mako-1.0.7-py3-none-any.whl
+99d041a616a3655725dffe459916627b75640a7d045ed33f51ce158a168ca3ef  Mako-1.0.7-py3-none-any.whl
 4e02fde57bd4abb5ec400181e4c314f56ac3e49ba4fb8b0d50bba18cb27d25ae  Mako-1.0.7.tar.gz
 c6b726d2e9d6300a044cf6a37627f10994268d6ac39464bc0d725126609311a5  MarkupSafe-1.0-py3-none-any.whl
```

Comment:
can this be removed?

Reply:
> can 1.0 entries be removed?

I think we should do it later with a new PR as we mess up, it will difficult to figure out where it broke (if it is the new work or removing old things).

Reply:
Oh, I guess you are talking about removing the old MarkupSafe wheel removal, that can be done.